examples/mqttc/Kconfig

```diff
@@ -0,0 +1,31 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config EXAMPLES_MQTTC
+	tristate "Enable MQTT-C Example"
+	default n
+  	depends on NETUTILS_MQTTC
+	---help---
+		Enable a simple MQTT-C publisher example
+
+if EXAMPLES_MQTTC
+
+config EXAMPLES_MQTTC_PROGNAME
+	string "Program name"
+	default "mqttc_pub"
+
+config EXAMPLES_MQTTC_STACKSIZE
+	int "Task's stack size"
+	default 8192
```

Comment:
Can we use the default stack size(2KB) here?

Reply:
I'm not sure if that's enough.  I remember that I had to adjust the stack size a while back when I had this example.  I'll do some tests later today and see if 2KB is enough.